src/libs/portfolio/portfolio.ts

```diff
@@ -29,6 +29,14 @@ import {
 } from './interfaces'
 import { flattenResults, paginate } from './pagination'
 
+// List of tokens to exclude from display by chainId and address
+const EXCLUDED_TOKENS: Array<{ chainId: bigint; address: string }> = [
+  { chainId: BigInt('100'), address: '0xcB444e90D8198415266c6a2724b7900fb12FC56E' }, // Gnosis
+  { chainId: BigInt('137'), address: '0x18ec0A6E18E5bc3784fDd3a3634b31245ab704F6' }, // Polygon
+  { chainId: BigInt('1'), address: '0x3231Cb76718CDeF2155FC47b5286d82e6eDA273f' }, // Ethereum
+  { chainId: BigInt('999'), address: '0x94e8396e0869c9F2200760aF0621aFd240E1CF38' } // Hyper EVM
```

Comment:
Imo this should be an object with an array of addresses to make it much faster and easier to search

{
   [chainId]: ['addr', 'addr2'],
   [chainId2]: ['addr3']
}

Reply:
Also, instead of comments for the networks, comment the symbol/name of the token and why it's excluded (like it's done in `getBannedToValidAddresses` in swap and bridge)